content/ch-gates/proving-universality.ipynb

```diff
@@ -388,7 +388,12 @@
     "\n",
     "Methods of this form can reproduce any unitary $U = e^{iH}$ for which $H$ can be expressed as a sum of tensor products of Paulis. Since we have shown previously that all matrices can be expressed in this way, this is sufficient to show that we can reproduce all unitaries. Though other methods may be better in practice, the main concept to take away from this chapter is that there is certainly a way to reproduce all multi-qubit unitaries using only the basic operations found in Qiskit. Quantum universality can be achieved!\n",
     "\n",
-    "This gate set is not the only one that can acheive universality. For example it can be shown that just the Hadamard and Toffoli are sufficient for universality. Multiple other gates sets have also been considered and been proven universal, each motivated by different routes toward acheiving the gates fault-tolerantly."
+    "This gate set is not the only one that can acheive universality. For example it can be shown that just the Hadamard and Toffoli are sufficient for universality. Multiple other gates sets have also been considered and been proven universal, each motivated by different routes toward acheiving the gates fault-tolerantly.\n",
+    "\n",
+    "Everything we have discussed in this chapter and most of this book follows the circuit model of computation. However, the circuit model is not the only universal model of quantum computation. There exists other forms of quantum computing such as the adiabatic quantum computing or the measurement based quantum computer. This means that it has been proven that there is a polynomial time and resource mapping from the circuit model to these models of computing.These often leverage other quantum mechanical properties also in order to perform their computation. However we do not need to delve these other forms for our purposes.\n",
```

Comment:
> ...most of this book...

I would just say "this book" as I believe all of the textbook uses gate based QC

> This means that it has been proven...

I would say "The fact that they are universal means it has been proven..."

> However we do not need to delve these...

Should make it clear to the reader the reason we have mentioned this. I imagine something like "It is important to be aware that these other forms of quantum computation exist, and that the benefits of each are hardware related. Since a universal model of quantum computation can perform any quantum algorithm, we need only stick with the gate model and can ignore other universal models."